src/main/java/com/github/alina/session_17_generics/challenge/xml_to_json/Convertor.java

```diff
@@ -0,0 +1,21 @@
+package com.github.alina.session_17_generics.challenge.xml_to_json;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+
+import java.io.File;
+import java.io.IOException;
+
+public class Convertor {
+    public static void main(String[] args) throws IOException {
+        XmlMapper mapper = new XmlMapper();
+        XmlMapper xmlMapper = new XmlMapper();
+        File file = new File("/Users/alina/Desktop/JavaCourseStartSteps/ExerciseAlgoritm/itschool1/src/main/java/com/github/alina/session_17_generics/challenge/xml_to_json/library.xml");
+        BookStore bookStore = xmlMapper.readValue(file, BookStore.class);
+        System.out.println(bookStore);
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.writeValue(new File("/Users/alina/Desktop/JavaCourseStartSteps/ExerciseAlgoritm/itschool1/src/main/java/com/github/alina/session_17_generics/challenge/xml_to_json/library.json"), bookStore);
```

Comment:
Please consider to use relative paths when possible. Our application, at least in the most cases, with consume/generate files inside of the project structure, and in this case, our application should not rely or be aware of the OS paths.
You can also make use of Path class:
`Path path = Paths.get("src/main/resources/library.json");`

Reply:
Hey, was working your suggestion. Thanks.